src/git-sync

```diff
@@ -26,7 +29,11 @@ function preconditions {
   if [ "$1" = "--all" ]; then
     sync_all=true
   elif [ "$(is_feature_branch "$INITIAL_BRANCH_NAME")" = false ]; then
+    # syncing a non-feature branch
```

Comment:
Alternatively we can add a couple more variables to make things a bit more clear and dry things up
```bash
if [ "$1" = "--all" ]; then
  sync_all=true
elif [ "$(is_feature_branch "$INITIAL_BRANCH_NAME")" = true ]; then
  syncing_feature_branch=true
else
  syncing_non_feature_branch=true
fi
```
And then use those variables instead of checking the same conditions elsewhere